horace_core/sqw/PixelData/@PixelData/PixelData.m

```diff
@@ -103,8 +103,9 @@
 end
 
 properties (Constant)
-    DEFAULT_NUM_PIX_FIELDS = 9;
     DATA_POINT_SIZE = 8;  % num bytes in a float
+    DEFAULT_NUM_PIX_FIELDS = 9;
+    DEFAULT_PAGE_SIZE = realmax;  % this gives no paging by default
```

Comment:
We were using `floor(realmax)` previously. Is the existence of the fractional part important?

Reply:
Also I would expect that the page size will be a large integer. So floor emphasizes the integer. But if looking for a max real, this will be 10^38 or 10^308 depending on the default type for realmax. It means unfeasibly large for a memory integer but do we want that or do we want a realistic upper bound on the amount of memory available, say via intmax with a suitably large integer type?

Reply:
`floor(realmax) == realmax`, so there's no functional difference. 

I'm hesitant to use an integer type because of Matlab's type propagation: `int64_var + double_var = int46_var`. Meaning that whenever we do an operation with `DEFAULT_PAGE_SIZE` the result will be an integer type, which can result in some interesting bugs.